subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/DeprecationCreationPerformanceTest.groovy

```diff
@@ -17,9 +17,7 @@
 package org.gradle.performance.regression.corefeature
 
 import org.gradle.performance.AbstractCrossVersionPerformanceTest
-import spock.lang.Ignore
 
-@Ignore('https://github.com/gradle/gradle-private/issues/3202')
 class DeprecationCreationPerformanceTest extends AbstractCrossVersionPerformanceTest {
```

Comment:
You need to increase the minimum base version to a version including the new deprecation API.

Reply:
Ah, thanks for reminder.